CPS-0012/README.md

```diff
@@ -13,24 +14,68 @@ Created: 2023-11-27
 
 ## Abstract
 
-Cardano lacks a standardized query layer. This leads to suboptimal tooling, dApp and wallet architecture.
+Blockchains are notoriously bad databases, as storage is expensive and queries are complex.
+Query layers services abstract away the difficulties of indexing blockchains, offering builders standard API interfaces to access data. 
+
+Cardano's query layers lack standardization.
+This leads to suboptimal tooling, dApp and wallet architecture.
+
+### Acknowledgements
+
+<details>
+  <summary><strong>Workshop 1 - 2024-01-25</strong></summary>
+
+  We would like to thank those that contributed to the first Query Layer Standardization workshop hosted by The Wallets Working Group ([see shared drive with resources](https://drive.google.com/drive/folders/1baSYHfWJdUh5dwRkHjY7qnaufjuO8sP2?usp=sharing)).
+
+  Hosts:
+
+  - Ryan Williams
+  - Adam Dean
+
+  Participants:
+
+  - Dmang
+  - George APEX Pool
+  - Leo H
+  - Marcin Szamotulski
+  - Markus Gufler
+  - Matt Davis
+  - Matthieu Pizenberg
+  - Michael Chappell
+  - NEXUS Crypto
+  - Nick Cook
+  - Rhys Bartels-Waller
+  - Ruslan Dudin
+  - Torbjørn Løvseth Finnøy
+
+</details>
 
 ## Problem
 
+Cardano nodes offer limited options for querying chain data.
+For builders, running chain indexer infrastructure can alleviate the data availability issues of the Cardano node.
+But this brings with it large overheads in running the nodes, chain followers, data storage and data access.
+
+Query layer providers offer builders the opportunity to abstract away the infrastructure complexities of Cardano data availability, usually to some cost.
+These providers are extremely useful and in turn are used as a foundational element for hundreds of tools.
+
+Cardano has a range of query layer providers, each with their own interfaces and business motivations.
+This can present issues for builders who wish to build with multiple providers.
+
 ### Query Layers and Tooling
 
 Lack of a standardized query layer results in multiple different implementations of roughly the same set of functionality:
 
 - [Blockfrost](https://blockfrost.io/)
 - [Koios](https://www.koios.rest/)
 - [Maestro](https://www.gomaestro.org)
-- [Ogmios](https://ogmios.dev/)
```

Comment:
Why not Ogmios?

Reply:
I dont think Ogmios really applies as it doesnt support historical data

Reply:
It is not a requirement for a query layer.

Reply:
yeah fair point

Reply:
changed!